frontend/packages/console-dynamic-plugin-sdk/docs/console-extensions.md

```diff
@@ -745,7 +745,7 @@ Adds new resource details page to Console router.
 | Name | Value Type | Optional | Description |
 | ---- | ---------- | -------- | ----------- |
 | `model` | `ExtensionK8sGroupKindModel` | no | The model for which this resource page links to. |
-| `component` | `CodeRef<React.ComponentType<{ match: match<{}>; namespace: string; model: ExtensionK8sModel; }>>` | no | The component to be rendered when the route matches. |
```

Comment:
More breaking changes...

Reply:
We discussed this with @vojtechszocs and its not actually a breaking change.